app/docs/0.12.x/loadbalancing.md

```diff
@@ -87,10 +76,36 @@ The DNS resolver will start resolving the following record types in order:
   3. A record
   4. CNAME record
 
-So if the hostname you use has both SRV entries and A entries, it will start
-with SRV. If you want A records to be used, you must remove the SRV records from
-the DNS server. If you only have A records, then the SRV lookup will fail and
-it will fallback on an A query, etc.
+This order is configurable through the [`dns_order` configuration property][dns-order-config].
+
+[Back to TOC](#table-of-contents)
+
+#### **DNS caveats**
+
+- Whenever the DNS record is refreshed a list is generated to handle the
+weighting properly. Try to keep the weights as multiples of each other to keep
+the algorithm performant, e.g., 2 weights of 17 and 31 would result in a structure 
+with 527 entries, whereas weights 16 and 32 (or their smallest relative 
```

Comment:
Please, for the sake of us, configure your editor to strip those trailing whitespaces. They make everyone else's diffs noisier than they should be and just bring conflict. See: https://github.com/Kong/kong/pull/3083

Reply:
@thibaultcha: just a side note, in this repo's [.editorconfig](https://github.com/Kong/getkong.org/blob/master/.editorconfig), `trim_trailing_whitespace` is explicitly set to `false` for `.md` files. (IIRC trailing whitespaces _are_ significant for Markdown, e.g. when marking code blocks via indentation and those contain indented blank lines.) 

So the best advice here is that everyone should use [editorconfig](https://editorconfig.org)-enabled editors!

Reply:
This is true but not the reason why `trim_trailing_whitespace` is disabled in this repository. Added in #374, the goal of the PR was to produce an editor config that would validate the current state of this source, nothing more.
Afaik, there are no significant trailing whitespaces in this entire website. Being its biggest contributor, and always trimming those, I never encountered any issue with it.
Once again, we are creating a toxic feedback loop over ridiculous problems.